client/src/pages/home/Home.jsx

```diff
@@ -0,0 +1,136 @@
+import "./home.css";
+import { useState } from "react";
+import Navbar from "../../components/navbar/Navbar.jsx";
+import MainContainer from "../../components/mainContainer/MainContainer.jsx";
+import Footer from "../../components/footer/Footer.jsx";
+import DoctorCard from "../../components/doctorComponents/doctorCard/DoctorCard.jsx";
+import Search from "../../components/search/Search.jsx";
+
+function Home() {
+  // NOTE: doc1, doc2 are temporary for front-end development purposes
+  // , use database entries as objects to pass in "users"
+
+  const doc1 = {
+    doctorName: "Kalameet The Dragon",
+    clinicAddress: "Rotripur-2301 New Delhi",
+    onlineConsulation: true,
+    treatments: [
+      "Wisdom tooth removal",
+      "Plaq Cleaning",
+      " Wonder tooth removal",
+      "tendon surgery",
+      "bone lengthening surgery",
+      "plastic surgery",
+    ],
+    specialization: [
+      "Surgeries",
+      "Allergy & Clinical Immunology",
+      "Anesthesiology",
+      "Community Medicine/Public Health",
+      "Dermatology, Venereology and Leprosy",
+    ],
+    consultationFee: 9000,
+  };
+
+  const doc2 = {
+    doctorName: "Sekiro",
+    clinicAddress: "Sempo Temple Column - 98123 ",
+    onlineConsulation: false,
+    treatments: ["Plastic Surgery"],
+    specialization: ["Surgeries", "Allergy Immunology"],
+    consultationFee: 700,
+  };
+  let doctorKey = 0;
+
+  const doctorObjects = [doc1, doc2];
+
+  // States for Search.jsx
+  const [SortOption, SwitchSortOption] = useState(false);
+  const [FilterOption, SwitchFilterOption] = useState(false);
+  const [SortByVal, ChangeSortBy] = useState("");
+
+  //States for FilterSearch.jsx
+  const [locationVal, locationChange] = useState("");
```

Comment:
Keep all state names consistent. Usually variable name is in camel case and dont include 'value' (sortBy, location) and then set<var name> (setSortBy, setLocation).

Reply:
I'm gonna solve all thing issues in the new version which I'm working on, I know it's the wrong thing, but I don't have much time to change it in this branch and the one I'm working on. Or maybe if I change it there and pull it might work but I am doing it in the easier way.